### PR TITLE
docs(design):  added docs for checkbox

### PR DESCRIPTION
**Pull request by Nolan-Arnold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Part of: #336 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x ] No
```

## Other information

### CODE REVIEW

File: libs/design/src/atoms/form/checkbox/checkbox.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# DaffCheckbox and DaffCheckboxSet
```
Comment (xelaint) [inline libs/design/src/atoms/form/checkbox/checkbox.md:1]: `Checkbox`

File: libs/design/src/atoms/form/checkbox/checkbox.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+The `DaffCheckboxComponent` is used to toggle an individual selection in a form. It can be hooked into Angular's `FormControl` to accomodate custom functionality. The `DaffCheckboxSetComponent` serves as a wrapper around a logical group of radios to provide styling.
+
+## Accessibility
+
+The `DaffCheckboxComponent` uses native `<input>` HTML elements to ensure an accesible experience by default.
+
+## DaffCheckbox Inputs
```
Comment (xelaint) [inline libs/design/src/atoms/form/checkbox/checkbox.md:8]: Can we combined `DaffCheckbox Inputs` and `DaffCheckbox Outputs` into one section called `DaffCheckbox Properties`.

Similarly, `DaffCheckboxSet Inputs` and `DaffCheckboxSet Functions` into one section called `DaffCheckboxSet Properties`?

Or are we purposely trying to create separation between inputs, outputs, and other functions?

File: libs/design/src/atoms/form/radio/radio.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# DaffRadio and DaffRadioSet
```
Comment (xelaint) [inline libs/design/src/atoms/form/radio/radio.md:1]: `Radio`

File: libs/design/src/atoms/form/radio/radio.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+The `DaffRadioComponent` is used to select a single value from a selection of values. It can be hooked into Angular's `FormControl` to accomodate custom functionality. The `DaffRadioSetComponent` serves as a wrapper around a logical group of radios to provide styling.
+
+## Accessibility
+The `DaffRadioComponent` uses native `<input>` HTML elements to ensure an accesible experience by default. It supports inputs to customize the experience for accessibility by allowing native input for `aria-label` and `aria-labelledby`.
+
+## DaffRadio Inputs
```
Comment (xelaint) [inline libs/design/src/atoms/form/radio/radio.md:7]: Same comment/question as checkbox.

File: libs/design/src/atoms/form/checkbox/checkbox.md (hunk 2 above)
Comment (Nolan-Arnold) [reply libs/design/src/atoms/form/checkbox/checkbox.md:8]: I like this idea a lot, no intent to separate them.  I think we could add just an extra column to the table called something like `property` or `property type` and fill it in with `input`, `output`, and `function`.

Comment (Nolan-Arnold) [reply libs/design/src/atoms/form/checkbox/checkbox.md]: updated in 73d1c0fef1a29a953f3ee8fc2da0b0801e61cc0c

File: libs/design/src/atoms/form/checkbox/checkbox.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+# Checkbox and CheckboxSet
+
+The `DaffCheckboxComponent` is used to toggle an individual selection in a form. It can be hooked into Angular's `FormControl` to accomodate custom functionality. The `DaffCheckboxSetComponent` serves as a wrapper around a logical group of radios to provide styling.
+
+## Accessibility
+
+The `DaffCheckboxComponent` uses native `<input>` HTML elements to ensure an accesible experience by default.
+
+## DaffCheckbox Properties
```
Comment (xelaint) [inline libs/design/src/atoms/form/checkbox/checkbox.md:9]: `Checkbox Properties`

File: libs/design/src/atoms/form/checkbox/checkbox.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+Property | Name            |  Type   |                                                                                         Description
+-------- | --------------- | :-----: | --------------------------------------------------------------------------------------------------:
+Input    | checked         | boolean |                 The value representIng whether the checkbox is checked. default is equal to `false` |
+Input    | value           | string  |                                                                           The value of the checkbox |
+Input    | id              | string  | The id of the checkbox. It is uniquely generated but can be overwritten by the user. Must be unique |
+Input    | name            | string  |                                                                            The name of the checkbox |
+Output   | becameChecked   | boolean |                                        An event that emits when the `checked` input becomes `true`. |
+Output   | becameUnchecked | boolean |                                       An event that emits when the `checked` input becomes `false`. |
+
+## DaffCheckboxSet Properties
```
Comment (xelaint) [inline libs/design/src/atoms/form/checkbox/checkbox.md:20]: `Checkbox Set Properties`

File: libs/design/src/atoms/form/radio/radio.md (hunk 4 above)
Comment (xelaint) [reply libs/design/src/atoms/form/radio/radio.md:7]: `Radio Properties`

File: libs/design/src/atoms/form/radio/radio.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+Input | checked | boolean | The value representing whether the radio is checked. default is equal to `false`|
+Input | value   | string  | The value of the radio | 
+Input |  id | string | The id of the radio. It is uniquely generated but can be overwritten by the user. Must be unique |
+Input |  name | string | The name of the radio |
+Input |  label | string | The label for the radio. Used for setting `aria-label` attribute. Defaults to equal name if nothing is provided. |
+Output | selectionChange | boolean | An event that emits everytime a selection has been changed.|
+
+
+
+## DaffRadioSet Properties
```
Comment (xelaint) [inline libs/design/src/atoms/form/radio/radio.md:20]: `Radio Set Properties`

**Comment by xelaint** [comment]
Additionally, I think these should be split into two PRs.

**Comment by Nolan-Arnold** [comment]
> Additionally, I think these should be split into two PRs.

Okay split into checkbox PR here, and radio here https://github.com/graycoreio/daffodil/pull/998.

File: libs/design/src/atoms/form/checkbox/checkbox.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
+# Checkbox and CheckboxSet
```
Comment (xelaint) [inline libs/design/src/atoms/form/checkbox/checkbox.md:1]: `Checkbox Set`

Comment (Nolan-Arnold) [reply libs/design/src/atoms/form/checkbox/checkbox.md]: Okay fixed in 20bd52cc779295676accbc64767be76fb8cda483

**Comment by xelaint** [comment]
@Nolan-Arnold there are still a few changes that need to be made from my review a few days ago.

**Comment by xelaint** [comment]
@Nolan-Arnold bumping this for updates.

**Comment by xelaint** [comment]
@Nolan-Arnold bumping this. There are few change requests that haven't been addressed yet.

**Comment by damienwebdev** [comment]
Closing as this is hella stale.

**Pull request closed without merging**